### PR TITLE
Product Inventory system service

### PR DESCRIPTION
**Pull request by SetMyDream**
Added Product model, form, DAO and service layer, routes and view.

### CODE REVIEW

File: product_inventory_service/build.sbt [hunk 1]
```diff
@@ -1,11 +1,72 @@
...
-    "org.http4s" %% "http4s-dsl" % http4sVersion,
-    "org.http4s" %% "http4s-blaze-server" % http4sVersion
+version := "1.0-SNAPSHOT"
+val AkkaVersion = "2.5.31"
+lazy val root = (project in file("."))
+  .enablePlugins(PlayScala, DockerPlugin)
+  .settings(
+    name := """play-scala-slick-example""",
+    version := "2.8.x",
+    scalaVersion := "2.13.4",
```
Comment (gemelen) [inline product_inventory_service/build.sbt:10]: Scala version №1

File: product_inventory_service/build.sbt [hunk 2]
```diff
@@ -1,11 +1,72 @@
...
+    scalacOptions ++= Seq(
+      "-feature",
+      "-deprecation",
+      "-Xfatal-warnings"
+    )
   )
-}
\ No newline at end of file
+
+scalaVersion := "2.12.3"
```
Comment (gemelen) [inline product_inventory_service/build.sbt:28]: Scala version №2.
Please decide which one to use and set it once

File: product_inventory_service/plugins.sbt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// The Play plugin
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.7")
+
+// The sbt native packager - needed for docker builds
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.8.0")
+
+// For code coverage test
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+
+// For checkstyle
```
Comment (gemelen) [inline product_inventory_service/plugins.sbt:10]: Please use `addSbtPlugin("com.beautiful-scala" % "sbt-scalastyle" % "1.5.0")` instead.

File: product_inventory_service/plugins.sbt [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
+
+// For formatting Scala source code
+addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.16")
```
Comment (gemelen) [inline product_inventory_service/plugins.sbt:14]: WTF is this?
`addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")` is official one

File: product_inventory_service/project/plugins.sbt [hunk 5]
```diff
@@ -0,0 +1,17 @@
+// The Play plugin
```
Comment (gemelen) [inline product_inventory_service/project/plugins.sbt:1]: You don't need this file here.

File: product_inventory_service/src/main/scala/conf/Application.conf [hunk 6]
```diff
@@ -0,0 +1,9 @@
+play.filters.disabled+=play.filters.csrf.CSRFFilter
```
Comment (gemelen) [inline product_inventory_service/src/main/scala/conf/Application.conf:1]: I believe it should be named `application.conf`

**Review by gemelen — CHANGES_REQUESTED** [review]

**Review by gemelen — APPROVED** [review]

**Review by me-anton — APPROVED** [review]

**Pull request merged**